R/cfbd_stats.R

```diff
@@ -198,7 +200,7 @@ cfbd_stats_game_advanced <- function(year,
                                      team = NULL,
                                      opponent = NULL,
                                      excl_garbage_time = FALSE,
-                                     season_type = "both") {
+                                     season_type = "regular") {
```

Comment:
why go from `both` to `regular` here? seems like everywhere else we're going from `regular` to `both`?